src/_gettsim/transfers/rente.py

```diff
@@ -19,8 +19,104 @@ def sum_ges_rente_priv_rente_m(priv_rente_m: float, ges_rente_m: float) -> float
     return out
 
 
-@policy_info(params_key_for_rounding="ges_rente")
-def ges_rente_vor_grundr_m(
+@policy_info(end_date="2020-12-31")
+def ges_rente_m(ges_rente_vor_grundr_m: float) -> float:
+    return ges_rente_vor_grundr_m
+
+
+@policy_info(
+    start_date="2021-01-01",
+    name_in_dag="ges_rente_m",
+    params_key_for_rounding="ges_rente",
+)
+def ges_rente_m_nach_grundr(
```

Comment:
```suggestion
def ges_rente_m(
```

Reply:
This doesn't work because we already have another function `ges_rente_m`. I chose `ges_rente_mit_grundrente_m` for now.

Reply:
I cannot look into it right now, but I am fairly sure the way I put it in the morning should with. No overlapping `ges_rente_m` at any point in time, that is.